Task 1: Hypothesis Statements

H(0): mu <= 1000 + 5(600): mu <= 4000: The mean weekly operating cost follows the theoretical model.

H(1): mu > 4000: The mean weekly operating cost is greater than the theoretical model.

x = 3050, mu = 4000, std = 5*25 =125, n = 25

In [1]:
sample_mean = 3050
theoretical_mean = 4000
sigma = 5*25
n = 25
alpha = 0.05

In [2]:
import numpy as np
import scipy.stats as stats

In [3]:
t_stat = (sample_mean - theoretical_mean)/(sigma / np.sqrt(n))
t_stat

np.float64(-38.0)

In [4]:
z_critical = stats.norm.ppf(1 - alpha)
z_critical

np.float64(1.644853626951472)

t_stat = -38

Z-critical = 1.645

t_stat < z-critical: Accept Null Hypothesis (H0)

##### Since we fail to reject the null hypothesis  H_0 , there is no strong statistical evidence that the actual weekly operating costs are higher than the theoretical cost model suggests.

##### This contradicts the restaurant owners’ claim that costs have increased. However, this test does not provide conclusive evidence that the costs have significantly decreased.